package.json

```diff
@@ -3,28 +3,43 @@
   "license": "MIT",
   "version": "0.1.0",
   "private": false,
+  "main": "dist/index",
+  "files": [
+    "contracts",
+    "dist"
+  ],
   "scripts": {
-    "build": "hardhat compile",
-    "lint": "eslint . --ext .js && solhint ./**/*.sol",
-    "prettier": "prettier --write \"./**/*.{json,js,md,sol}\"",
+    "install": "yarn build",
```

Comment:
We need the types, which are generated during build. I've tried many different combinations and this works. This is to allow the package to be imported via git rather than via npm.

Reply:
`build` is not normally called as part of the yarn lifecycle scripts: https://yarnpkg.com/advanced/lifecycle-scripts
So we have to hook into `install` or commit the built files. This is assuming we want to support git as an import, which I think we do during initial dev.

Reply:
I see. Makes sense.